pkg/login/other_logins_check.go

```diff
@@ -0,0 +1,151 @@
+package login
```

Comment:
Thanks, @iamkirkbater, for the PR. What if we change the file name to something readable like "user-sesseion-check" . My main thinking is `other-login-checks` is too generic and may not isolate the real use of it, In future, there may be more of these types of checks, and we could modularize it if we have dedicated `check-name`

+1 for adding the unit tests..

Reply:
Names are 1000% fungible here. I like the suggestions. I got hung up on figuring out a name so I just named it something stupid to move on, I like your suggestions better :)